using-mobile-extensions/adobe-campaign-standard/README.md

```diff
@@ -1,16 +1,16 @@
 # Adobe Campaign Standard
 
 {% hint style="info" %}
-**Before** you install or configure the Campaign Standard extension, read [_Getting Started_](../../getting-started/create-a-mobile-property.md) and [Configuring a mobile application using Adobe Experience Platform SDKs](https://helpx.adobe.com/campaign/kb/configuring-app-sdk.html).
+**Before** you install or configure the Campaign Standard extension, please read the [getting started guide](../../getting-started/create-a-mobile-property.md) and the [configuring a mobile application using Adobe Experience Platform SDKs guide](https://experienceleague.adobe.com/docs/campaign-standard/using/administrating/configuring-channels/configuring-a-mobile-application.html).
 {% endhint %}
 
 {% hint style="danger" %}
-If you participated in the Campaign Standard beta, to use the new Campaign Standard extension, go to [launch.adobe.com](https://launch.adobe.com), instead of the Experience Platform Launch integration environment, .
+If you participated in the Campaign Standard beta, to use the new Campaign Standard extension, go to the [Adobe Experience Platform Data Collection UI](https://experience.adobe.com/data-collection/), instead of the Experience Platform Data Collection integration environment.
 {% endhint %}
 
-## Configure the Campaign Standard extension in Experience Platform Launch
+## Configure the Campaign Standard extension in Experience Platform Data Collection
 
-1. In Experience Platform Launch, click the **Extensions** tab.
+1. In the Experience Platform Data Collection UI, click the **Extensions** tab.
```

Comment:
Drop "Experience Platform". There are a few other places in this doc where "Data Collection UI" needs the same correction.